packages/components/src/components/icon/test/html.mock.ts

```diff
@@ -1,19 +1,24 @@
 import { mixMembers } from 'stencil-awesome-test';
+import path from 'path';
+import { readFileSync } from 'fs';
+import { render } from 'mustache';
 
 import { KoliBriIconProps } from '../types';
 
 export const getIconHtml = (props: KoliBriIconProps, additionalAttrs = ''): string => {
-	props = mixMembers(
+	const state = mixMembers(
 		{
 			_icon: 'codicon codicon-home',
 		},
 		props
 	);
-	return `<kol-icon exportparts="icon"${additionalAttrs}>
-  <mock:shadow-root>
-    <i ${typeof props._label === 'string' && props._label.length > 0 ? `aria-label="${props._label}"` : 'aria-hidden="true"'} class="${
-		props._icon
-	}" part="icon" role="img"></i>
-  </mock:shadow-root> 
-</kol-icon>`;
+
+	const template = readFileSync(path.join(__dirname, 'icon.mustache'), { encoding: 'utf-8' });
```

Comment:
Mustache scheint nicht auf Node.js ausgelegt zu sein: Wir müssen das Template selber laden.

Als Problem sehe ich auch, dass Änderungen im Template keinen neuen Test-Durchlauf in Jest im watch mode auslösen.